patterns/builder.md

```diff
@@ -8,38 +8,53 @@ Construct an object with calls to a builder helper.
 ## Example
 
 ```rust
-struct Foo {
+use std::fmt;
+
+pub struct Foo {
     // Lots of complicated fields.
+    bar: String,
 }
 
-struct FooBuilder {
+// Optional, define the textual representation for Foo.
+impl fmt::Display for Foo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "bar: {}", self.bar)
+    }
+}
```

Comment:
This `Display` impl feels irrelevant to illustrating the builder pattern. Can we replace this with `#[derive(Debug)]`, and use `format!("{:?}", foo)` in the test instead?

Reply:
I will look for this soon 🙂

Reply:
Yeah you are right! Changed this up 👷